walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -42,6 +44,36 @@
     timeOptions = getPossibleTimes(occupiedTime, request);
     mandatoryOptions = getPossibleTimes(mandatoryTime, request);
     // Check if the returned times work for all mandatory and optional attendees
+    if (timeOptions.isEmpty() && !mandatoryOptions.isEmpty() && request.getOptionalAttendees().size() > 1) {  
+      List<TimeRange> mandatoryTime2 = new ArrayList<TimeRange>();
+      Collection<TimeRange> optionsReserve = new ArrayList<TimeRange>();
+      // Store a reserve of the original time options
+      optionsReserve = mandatoryOptions;
+      // Copy the occupied time slots into a new variable for manipulation 
+      mandatoryTime2 = mandatoryTime;
+      // Go through each event the optional attendees have and add it to the total
+      // occupied times. If the no time is returned, remove the input time. 
```

Comment:
"If no time is returned"?

Reply:
If there was a time slot that worked for the mandatory attendees but adding the all of the optional attendees causes no time to be returned.

Reply:
Clarified in commit ad8364a. Also added periods on each comment.